.github/workflows/release.yaml

```diff
@@ -213,26 +213,106 @@ jobs:
           category: ${{ matrix.flavor }}
   build-core-uki:
     runs-on: ubuntu-latest
-    needs:
-      - build-core
-      - get-uki-matrix
```

Comment:
this is still needed no? Otherwise the matrix below will fail?

Reply:
:man_facepalming: